src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/mbscheduler_hwemu.cxx

```diff
@@ -2222,6 +2222,67 @@ namespace hwemu {
         return 0;
     }
 
+    int xocl_scheduler::convert_execbuf(xocl_cmd* xcmd)
+    {
+        size_t src_off;
+        size_t dst_off;
+        size_t sz;
+        uint64_t src_addr = -1; 
+        uint64_t dst_addr = -1; 

```

Comment:
It would be great if these declarations were moved to the point where they are defined.